tools/AutomationTestFramework/ManageScripts/Main.ps1

```diff
@@ -0,0 +1,83 @@
+
+. "$PSScriptRoot\GenerateAndUploadTestHelperModule.ps1"
+. "$PSScriptRoot\GenerateAndUploadTestModule.ps1"
+. "$PSScriptRoot\RemoveModules.ps1"
+. "$PSScriptRoot\UploadSignedModules.ps1"
+. "$PSScriptRoot\GenerateRunbook.ps1"
+. "$PSScriptRoot\AccountRunbook.ps1"
+
+<#
+.SYNOPSIS
+Complete Automation Account TEST Framework scenario  
+.PARAMETER i_want_to
+Set of switches to enable or disable certain functionality
+.NOTES
+Prerequisite: Ready to use Azure Automation Account and Azure Storage Account container
+with settings provided in the AzureConfig.ps1 file
+#>
+function Main (
```

Comment:
We will normally be executing this from PowerShell, so the simplest thing is to make this entire file a script cmdlet, and take the parameters at the top, similar to these scripts:

This is maonly a matter of changing the name of the file to RunAutomationTests.ps1, and removing the function definition.https://github.com/Azure/azure-powershell/blob/preview/tools/PublishModules.ps1#L1

Reply:
all done.